From: Ken Thomases <ken@codeweavers.com>
Date: Fri, 21 Jan 2011 00:58:57 +0000 (-0600)
Subject: windowscodecs: Disable ICNS support when building for Mac OS X 10.4 Tiger.
X-Git-Tag: wine-1.3.12~27
X-Git-Url: http://source.winehq.org/git?hp=9c66f68f55beca6fc92315d40a73b41f12865353

windowscodecs: Disable ICNS support when building for Mac OS X 10.4 Tiger.
---

--- dlls/windowscodecs/icnsformat.c.orig
+++ dlls/windowscodecs/icnsformat.c
@@ -68,7 +68,8 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(wincodecs);
 
-#ifdef HAVE_APPLICATIONSERVICES_APPLICATIONSERVICES_H
+#if defined(HAVE_APPLICATIONSERVICES_APPLICATIONSERVICES_H) && \
+    MAC_OS_X_VERSION_MAX_ALLOWED > MAC_OS_X_VERSION_10_4
 
 typedef struct IcnsEncoder {
     IWICBitmapEncoder IWICBitmapEncoder_iface;
@@ -752,7 +753,8 @@ HRESULT IcnsEncoder_CreateInstance(IUnknown *pUnkOuter, REFIID iid, void** ppv)
     return ret;
 }
 
-#else /* !defined(HAVE_APPLICATIONSERVICES_APPLICATIONSERVICES_H) */
+#else /* !defined(HAVE_APPLICATIONSERVICES_APPLICATIONSERVICES_H) ||
+         MAC_OS_X_VERSION_MAX_ALLOWED <= MAC_OS_X_VERSION_10_4 */
 
 HRESULT IcnsEncoder_CreateInstance(IUnknown *pUnkOuter, REFIID iid, void** ppv)
 {
